apple/Sources/FerrostarMapLibreUI/Views/Overylays/LandscapeNavigationOverlayView.swift

```diff
@@ -11,6 +11,9 @@ struct LandscapeNavigationOverlayView: View, CustomizableNavigatingInnerGridView
 
     private var navigationState: NavigationState?
 
+    @State private var isInstructionViewExpanded: Bool = false
+    // TODO: Same as portrait?
```

Comment:
Did you intend to also make this state + add dynamic padding in landscape like you did for portrait?

Reply:
No - I don't think we need to. 

Here's a crude schematic explaining why:

Before this PR, in the portrait layout we used to have:
```
VStack {
  InstructionView
  GridView  // <- grid view is below the instruction view 
  ArrivalView
}
```

To allow the instructionView to expand *over top* of the other content, we now have:
```
ZStack(align: .top) {
  InstructionView
  VStack {
     // !!! We now need to add top padding to account for the InstructionView height
     // otherwise the GridView would be obscured behind the unexpanded instruction view
    GridView.padding(.top, XX_InstructionView.minimizedHeight_XX)
    ArrivalView
  }
}
```

However, things are easier with the landscape view. 
The GridView is in its own column, so it can't be covered by the ArrivalView.

Landscape:
```
HStack {
  VStack {
    InstructionView
    ArrivalView
  }
  // GridView is in its own column, so won't be covered by the InstructionView
  GridView 
}
```

Does that make sense and answer your question?

Reply:
Yes, perfectly! Thanks!